karapace/schema_registry.py

```diff
@@ -466,6 +466,19 @@ def send_schema_message(
             value = None
         self.producer.send_message(key=key, value=value)
 
+    def is_topic_requiring_validation(self, *, topic_name: TopicName) -> bool:
+        return self.database.is_topic_requiring_validation(topic_name=topic_name)
+
+    def update_require_validation_for_topic(
+        self,
+        *,
+        topic_name: TopicName,
+        skip_validation: bool,
+    ) -> None:
+        key = {"keytype": str(MessageType.schema_validation), "magic": 0}
+        value = {"skip_validation": skip_validation, "topic": topic_name}
+        self.producer.send_message(key=key, value=value)
```

Comment:
Depending on `skip_validation` this could also send tombstone record instead of toggling between true/false.

Reply:
I don't think so, otherwise how can you rollback from disabling the validation for a certain topic?